.github/CODEOWNERS

```diff
@@ -37,6 +37,12 @@
 # ServiceLabel: %Azure.Core
 # AzureSdkOwners:                                    @alzimmermsft
 
+# PRLabel: %Azure.Core.V2
+/sdk/corev2/                                         @alzimmermsft @srnagar @anuchandy @lmolkova @vcolin7 @mssfang @jonathangiles @billwert
```

Comment:
Could I get added here too :)

Reply:
I'll file a follow-up PR adding you to here and /sdk/core